source/config/__init__.py

```diff
@@ -914,6 +762,21 @@ def saveProfileTriggers(self):
 		self.triggersToProfiles.parent.write()
 		log.info("Profile triggers saved")
 
+	def getConfigValidationParameter(self, keyPath, validationParameter):
+		"""Get a config validation parameter
+		This can be used to get the min, max, default, or other values for a config key.
+		@param keyPath: a sequence of the identifiers leading to the config key. EG ("braille", "messageTimeout")
+		@param validationParameter: the parameter to return the value for. EG "max"
+		@type validationParameter: string
+		"""
+		if not keyPath or len(keyPath) < 1:
+			raise ValueError("Key path not provided")
+
+		spec = conf.spec
+		for nextKey in keyPath:
+			spec = spec[nextKey]
+		return conf.validator._parse_with_caching(spec)[2][validationParameter]

```

Comment:
What i mean here is that in parts of the config, we have the parameter 
`[[__many__]]`
and I don't see a way to get the min or max from those. Could this be triggering the bug that was mentioned on NVDA-devel? [1] and [2]

[1]: http://nabble.nvda-project.org/Re-nvda-next-6157f43e-Incubates-6558-td41486.html
[2]: http://nabble.nvda-project.org/Latest-next-seems-to-crash-on-restart-locking-up-computer-tp41490.html